integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/BlueprintTest.java

```diff
@@ -1,4 +1,4 @@
-package com.sequenceiq.it.cloudbreak.newway.testcase;
+package com.sequenceiq.it.cloudbreak.newway.testcase.mock;
```

Comment:
is it run on circle? Is the yaml pattern include the subpackage?

Reply:
I update it